### PR TITLE
Domains: Update notices for transfer

### PR DESCRIPTION
**Pull request by aidvu**
Add 3 cases when we initiate transfer:
- When we couldn't get the e-mail right away
- When we got the email, and waiting for the FOA to be sent
- When we already have the email

### CODE REVIEW

File: client/my-sites/domains/components/domain-warnings/index.jsx [hunk 1]
```diff
@@ -795,16 +795,34 @@ export class DomainWarnings extends React.PureComponent {
...
-						args: { domain: domainInTransfer.name },
-					}
-				);
+
+				const translateParams = {
+					components: {
+						strong: <strong />,
+						a: <a href={ domainManagementLink } rel="noopener noreferrer" />,
+					},
+					args: { domain: domainInTransfer.name },
```
Comment (danhauk) [inline client/my-sites/domains/components/domain-warnings/index.jsx]: Do we also need `email: domainInTransfer.email` here for the case where we have the email address to show?

Comment (aidvu) [reply client/my-sites/domains/components/domain-warnings/index.jsx]: Nope. I set it explicitly for that message https://github.com/Automattic/wp-calypso/pull/21470/files#diff-ef7444613272677858192c00d814e81cR814.

Comment (danhauk) [reply client/my-sites/domains/components/domain-warnings/index.jsx]: Ha, I guess I should've looked harder there. Nice work!

**Comment by danhauk** [comment]
LGTM! 👍

**Pull request merged**